src/main/webapp/scripts/session-script.js

```diff
@@ -1,46 +1,84 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/**
+ * An array of who is currently in the session.
+ * @type {Object}
+ */
+let currentAttendees = [];
 
 /**
- * function openSessionInfo() displays the div container
- * that has information about the session.
+ * Represents the current session, a Session object.
+ * @type {Object}
  */
-function openSessionInfo() {
-  document.getElementById('session-info-div').style.display = 'block'; 
-}
+let session;
+
+/**
+ * Represents (in miliseconds) how long users are alerted of any membership
+ * changes in the session. 
+ * @type {number}
+ */
+const DISPLAY_CADENCE = 4000;
```

Comment:
DISPLAY_CADENCE_MS

Always, always, always put the granularity of time at the end of a time variable when its not obvious so that people know what theyre looking at.